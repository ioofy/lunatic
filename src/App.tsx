const App = () => {
  return <div>I'm akwelakleklakelalw</div>;
};

export default App;
