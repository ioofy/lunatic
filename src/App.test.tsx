import { render, screen, waitFor } from "@testing-library/react";
import App from "./App";

describe("Home Screen", () => {
  it("Render Home screen", () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });

  it("check if Home screen renders", async () => {
    render(<App />);

    await waitFor(() => screen.getByText("App"));
    expect(screen.getByText("App").innerHTML).toBe("App");
  });
});
