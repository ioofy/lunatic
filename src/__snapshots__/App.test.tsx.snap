// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Screen Render Home screen 1`] = `
<div>
  <div>
    I'm akwelakleklakelalw
  </div>
</div>
`;
